session/session.go

```diff
@@ -274,7 +275,7 @@ type session struct {
 	idxUsageCollector *handle.SessionIndexUsageCollector
 
 	functionUsageMu struct {
-		sync.RWMutex
+		syncutil.RWMutex
```

Comment:
Do you have a plan for the rest of all other `sync.RWMutex` usage in TiDB?

Reply:
Yes, But deadlock detection will slow the test so I need to slowly transition to ```syncutil```.